.readme-partials.yml

```diff
@@ -80,20 +80,18 @@ body: |-
   If you already have a "raw" Http `request` object you can assign it to `entry.metadata.httpRequest` directly. More information about
   how the `request` is interpreted as raw can be found in the [code](https://github.com/googleapis/nodejs-logging/blob/15849160116a814ab71113138cb211c2e0c2d4b4/src/entry.ts#L224-L238).
 
-  ## Automatic Trace/Span Id Extraction
-  Cloud Logging libraries use [trace fields within LogEntry](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#FIELDS.trace) to capture trace contexts, which enables the [correlation of logs and traces](https://cloud.google.com/logging/docs/view/correlate-logs), and supporting distributed tracing. 
-  These trace fields, including [trace](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#FIELDS.trace), [spanId](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#FIELDS.span_id), and [traceSampled](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#FIELDS.trace_sampled) define the trace context for the log entry.
+  ## Automatic trace/span Id extraction
+  Cloud Logging libraries use [trace fields within LogEntry](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#FIELDS.trace) to capture trace contexts, which enables the [correlation of logs and traces](https://cloud.google.com/logging/docs/view/correlate-logs), and distributed tracing troubleshooting. 
+  These tracing fields including [trace](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#FIELDS.trace), [spanId](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#FIELDS.span_id), and [traceSampled](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#FIELDS.trace_sampled) define the trace context for a `LogEntry`.
 
-  If not provided explicitly, Google Cloud Logging library will automatically populate `LogEntry fields`_
-  `trace`, `span_id`, and `trace_sampled` from detected OpenTelemetry span context, or HTTP request headers.
+  If not provided explicitly in a LogEntry, Google Cloud Logging library will automatically populate `trace`, `span_id`, and `trace_sampled` fields from detected OpenTelemetry span contexts, or HTTP request headers.
 
-  ### Extracting from OpenTelemetry Context
+  ### Extracting trace/span Id from OpenTelemetry Context
   If you are using OpenTelemetry and there is an active span in the OpenTelemetry Context, that log entry will automatically have the `trace`, `span_id`, and `trace_sampled` fields populated from that span. More information about OpenTelemetry can be found [here](https://opentelemetry.io/docs/languages/js/).
```

Comment:
Starting with "that log entry" feels awkward to me (what log entry?). I think it would read better as
"the `trace`, `span_id`, and `trace_sampled` fields in the log entry are automatically populated from the active span."